src/DateHeader.js

```diff
@@ -1,9 +1,18 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 
-const DateHeader = ({ label, drilldownView, onDrillDown }) => {
+const DateHeader = ({ label, localizer, date, drilldownView, onDrillDown }) => {
+  const renderLabel = () => (
```

Comment:
Doing it this way recreates this function with each render of the DateHeader. Create a separate HeaderLabel component function above the DateHeader (you don't need to export it), passing it the `label`, `date` and `localizer`, and use the component within the DateHeader.

Reply:
Updated to move the label into a separate component.